sgkit_plink/cli.py

```diff
@@ -0,0 +1,95 @@
+"""
+Command line utilities for conversion.
+"""
+import argparse
+import os
+import signal
+
+from dask.diagnostics import ProgressBar
+
+from . import read_plink
+
+
+def set_sigpipe_handler():
+    if os.name == "posix":
+        # Set signal handler for SIGPIPE to quietly kill the program.
+        signal.signal(signal.SIGPIPE, signal.SIG_DFL)
+
+
+def run_plink2sg(args):
+    # TODO add args for bim and fam seps
+    # TODO add logging and verbosity so we can write out info messages
+    # about what we're doing here.
+    ds = read_plink(args.plink_path, bim_sep=" ", fam_sep=" ")
+
+    # TODO add a command line options for this progess bar.
+    with ProgressBar():
```

Comment:
Does this work well with `ds.to_zarr`?  Or is it kind an ambiguous mess where multiple of them kick off and it's unclear how many different progress bars there will be in the end?

Reply:
It worked fine, in my brief testing.